containers/python/rpc.py

```diff
@@ -212,6 +216,11 @@ def start(model, ip, port, model_name, model_version, input_type):
         model_version (int): The version of the model
         input_type (str): One of ints, doubles, floats, bytes, strings.
     """
+
+    try:
+        ip = socket.gethostbyname(host)
+    except socket.error as e:
+        print("Error resolving %s: %s" % (host, e))
```

Comment:
We should exit here via `sys.exit(1)`. Otherwise, we attempt to connect to a potentially invalid address.